CONTRIBUTING.md

```diff
@@ -59,32 +61,49 @@ All components are **responsive** by default, so there is no need for modifiers
 ## React
 
 Each component/container should have its own file.
-Components are dumb and should be written in pure function form. 
+Components are dumb and should be written in pure function form.
 Each file should export the default module like so:
+
 ```
 export default ComponentName;
 ```
 
 ### File Location
 
-Component and container files should be located next to `.scss` files in `src/components` directory. 
+Component and container files should be located next to `.scss` files in `src/components` directory.
 Documentation pages for components should be located in same directory in a subdirectory called `pages`.
 
 ### Tests
 
 Each component should be tested. We are using [jest](https://facebook.github.io/jest/) as a test runner/framework.
 Test files should be located next to component/container file with extension `.spec.js`
 
+### Storybook & Chromatic
+
+All components should have storybook stories. Put those stories in `{component_name}.stories.jsx` next to component.
+
+There is also Chromatic visual regression tool which uses storybook stories to run tests. It is integrated with Github and being run on pull requests.
+
+If you already have storybook stories, you can use them also for Chromatic. Run script:
+
+```
+$ node ./scripts/generate-chromatic-stories.js
+```
+
+which looks for `{component_name}.chromatic.stories.jsx` files and generates `{component_name}.chromatic.stories.jsx` for each of them(if not exists). Created files have single exported story(Default). You can extend it with your stories if needed.
```

Comment:
@clxandstuff did you mean `{component_name}.stories.jsx` in first part?

I would also rephrase: `You can extend it with your stories if needed.` with `You can extend it with chromatic specific stories if needed.`

Reply:
Done